_data/projects/microkit.yml

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: BSD-2-Clause
+# Copyright 2020 seL4 Project a Series of LF Projects, LLC.
+
+name: microkit
+display_name: Microkit
+description: A simple operating system framework for developing systems on top of seL4
+project_order: 10
+
+useful_urls:
+    - label: "Microkit user manual"
+      url: "https://github.com/seL4/microkit/blob/main/docs/manual.md"
+    - label: "Microkit tutorial"
+      url: "https://trustworthy.systems/projects/microkit/tutorial"
+    - label: "Microkit development roadmap"
+      url: "https://github.com/seL4/microkit/issues/61"
+    - label: "Trustworthy Systems Microkit project"
+      url: "https://trustworthy.systems/projects/microkit"
+
+repositories:
+  - org: sel4
+    repo: microkit
+  - org: au-ts
```

Comment:
I listed the repo for the Microkit tutorial here, but not sure if these links should be restricted to official seL4 projects.

Reply:
We should probably move the tutorial over to the seL4 org in any case, if it's closely linked to microkit. Are there any UNSW reasons not to move it? In the meantime, I think it's fine to have the link there.

Reply:
Result from offline discussion: we should move it, but not yet.